erpnext_documentation/www/docs/user/manual/en/human-resources/setting-up-tax.md

```diff
@@ -38,25 +38,25 @@ For the fiscal year 2018-19, in India, House Rent Allowance (HRA) exemption from
+* Deduct Tax For Unsubmitted Tax Exemption Proof: This option allows you to deduct taxes for the earnings which were exempted in previous payrolls as declared in [Employee Tax Exemption Declaration](/docs/user/manual/en/human-resources/employee-tax-exemption-declaration) but the Employee has not submitted sufficient proof via  [Employee Tax Exemption Proof Submission](/docs/user/manual/en/human-resources/employee-tax-exemption-proof-submission). It is to be noted that if this option is checked ERPNext does not consider the Employee Tax Exemption Declaration by employees and will only take into account _Employee Tax Exemption Proof Submission_ instead while calculating exemption from employees' annual earnings.
 
 >Note: If required, you can still process payroll for employees individually, by manually creating a new Salary Slip and both these options are made available in the Salary Slip
 
-#Payroll Period
-[Payroll Period](/docs/user/manual/en/human-resources/payroll-period) helps you define Tax slabs applicable for the period, making it easier to manage changing laws. You can add multiple tax slabs for the payroll period depending on the tax regulations. Note that you can use fields in Employee document in the _Condition_ field to apply tax slabs based on attributes of employees.
+# Income Tax Slab
+[Income Tax Slab](/docs/user/manual/en/human-resources/income-tax-slab) helps you define Tax slabs applicable for the period, making it easier to manage changing laws. You can add multiple tax slabs for the payroll period depending on the tax regulations. Note that you can use fields in Employee document in the _Condition_ field to apply tax slabs based on attributes of employees.
 
-#Salary Component
-To enable automatic tax deduction based on Tax slabs configured in Payroll Period, you have to configure a Salary Component of type _Deduction_ with _Variable Based On Taxable Salary_ option enabled. If you enable this, the component will be considered as the standard Tax deduction component and tax will be calculated based on the Tax slabs configured in Payroll Period on all the total taxable salary.
+# Salary Component
+To enable automatic tax deduction based on Tax slabs configured in Income Tax Slab, you have to configure a Salary Component of type _Deduction_ with _Variable Based On Taxable Salary_ option enabled. If you enable this, the component will be considered as the standard Income Tax component and tax will be calculated based on the Tax slabs configured in Income Tax Slab on all the total taxable salary.
```

Comment:
"If you enable this the component will be considered as the standard Income Tax component and tax will be calculated based on the Tax slabs configured in Income Tax Slab on all the total taxable salary."

to

"This checkbox enables auto calculation of Income Tax considering the tax slabs and declaration submitted by an employee. The tax will be calculated annually on the remaining taxable salary and equally divide it in 12 months."